### PR TITLE
chore(package.json): update angular.js to v1.9.3 along with other non-angularjs related dependencies

### PR DESCRIPTION
**Pull request by jzolnowski**
After executing `npm install`, `package-lock.json` was also updated to reflect the changes from `package.json`.
In the process of updating the lockfile, several versions of the dependencies were upgraded automatically.

### CODE REVIEW

**Review by gkalpak — CHANGES_REQUESTED** [review]
We should also update the lockfile.

File: package-lock.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {
   "name": "angularjs-material-vite",
   "version": "1.1.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```
Comment (gkalpak) [inline package-lock.json:4]: We should not be updating the lockfile version in this commit.

**Review by gkalpak — CHANGES_REQUESTED** [review]
This PR now contains more updates than just "angular.js to v1.9.2".
We should either limit the changes to AngularJS or update the commit message to reflect the actual changes.

**Review by gkalpak — CHANGES_REQUESTED** [review]
The changes lgtm, but I find the commit message confusing/unclear. What does _"angular.js-based libraries"_ refer to?

**Comment by jzolnowski** [comment]
> The changes lgtm, but I find the commit message confusing/unclear. What does _"angular.js-based libraries"_ refer to?

angular.js-based librarie means all the rest bumped angularjs-based packages like:

```
"angular-animate": "npm:@xlts.dev/angular-animate@^1.9.2",
"angular-aria": "npm:@xlts.dev/angular-aria@^1.9.2",
"angular-material": "npm:@xlts.dev/angular-material@^1.2.4",
"angular-messages": "npm:@xlts.dev/angular-messages@^1.9.2"
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
>> The changes lgtm, but I find the commit message confusing/unclear. What does _"angular.js-based libraries"_ refer to?
>
> angular.js-based librarie means all the rest bumped angularjs-based packages like:
>
> ```
> "angular-animate": "npm:@xlts.dev/angular-animate@^1.9.2",
> "angular-aria": "npm:@xlts.dev/angular-aria@^1.9.2",
> "angular-material": "npm:@xlts.dev/angular-material@^1.2.4",
> "angular-messages": "npm:@xlts.dev/angular-messages@^1.9.2"
> ```

I see. I don't think we need to explicitly mention these libraries in the commit message, since they are part of the AngularJS framework and can be collectively referred to as AngularJS. 

But this PR also updates other non-AngularJS related dependencies. We should either mention that in the commit message to move those other updates to a separate commit/PR.

**Review by gkalpak — CHANGES_REQUESTED** [review]
We should now update to v1.9.3 :wink:

**Comment by jzolnowski** [comment]
> We should now update to v1.9.3 😉

Done

**Review by gkalpak — APPROVED** [review]

**Pull request merged**